pkg/rancher-desktop/main/extensions/manager.ts

```diff
@@ -141,8 +141,6 @@ export class ExtensionManagerImpl implements ExtensionManager {
       case 'container':
         return this.execStreaming(event, options, await this.spawnContainer(event, options));
       }
-      console.error(`Unexpected scope ${ options.scope }`);
-      throw new Error(`Unexpected scope ${ options.scope }`);
```

Comment:
We're no longer protecting against accidentally reaching should-be-unreachable code?